redish/libs/redish-frontend/authentication/ui/src/lib/register-user-form/register-user-form.tsx

```diff
@@ -27,14 +35,19 @@ export function CreateUserForm(props: { viewModel: CreateUserFormViewModel }) {
           Choose password:
           <input
             type="password"
+            autoComplete="new-password"
+            required
             value={viewModel.registerUser.password}
             onChange={(e) => viewModel.handlePasswordChanged(e.target.value)}
           />
         </label>
         <input type="submit" />
       </form>
+      <p>
+        already have a user? <Link to={viewModel.loginRoute}>Login here</Link>
```

Comment:
This should probably say something like "already have an account" or "already registered"

Reply:
👌